invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/search.less

```
@@ -26,3 +26,13 @@
 .identifiers {
   display: inline;
 }
+
+.ui.items > .item .extra.labels-actions {
```

add a short comment for what its done
```suggestion
// align badges to the right
.ui.items > .item .extra.labels-actions {
```